ogzaf/demographics.py

```diff
@@ -1,192 +1,488 @@
+    # UN variable code for Population by 1-year age groups and sex
+    pop_code = "47"
+    # UN variable code for Fertility rates by age of mother (1-year)
+    fert_code = "68"
+
+    if download:
+        pop_target = (
+            "https://population.un.org/dataportalapi/api/v1/data/indicators/"
+            + pop_code
+            + "/locations/"
+            + country_id
+            + "/start/"
+            + str(start_year)
+            + "/end/"
+            + str(end_year)
+            + "?format=csv"
+        )
+        fert_target = (
+            "https://population.un.org/dataportalapi/api/v1/data/indicators/"
+            + fert_code
+            + "/locations/"
+            + country_id
+            + "/start/"
+            + str(start_year)
+            + "/end/"
+            + str(end_year)
+            + "?format=csv"
+        )
+    else:
+        pop_target = os.path.join(DATA_DIR, "un_zaf_pop.csv")
+        fert_target = os.path.join(DATA_DIR, "un_zaf_fert.csv")
+
+    # Convert .csv file to Pandas DataFrame
+    pop_df = pd.read_csv(
+        pop_target,
+        sep="|",
+        header=1,
+        usecols=["TimeLabel", "SexId", "Sex", "AgeStart", "Value"],
+        float_precision="round_trip",
+    )
+    fert_rates_df = pd.read_csv(
+        fert_target,
+        sep="|",
+        header=1,
+        usecols=["TimeLabel", "AgeStart", "Value"],
+        float_precision="round_trip",
+    )
+
+    # Rename variables in the population and fertility rates data
+    pop_df.rename(
+        columns={
+            "TimeLabel": "year",
+            "SexId": "sex_num",
+            "Sex": "sex_str",
+            "AgeStart": "age",
+            "Value": "pop",
+        },
+        inplace=True,
+    )
+    fert_rates_df.rename(
+        columns={
+            "TimeLabel": "year",
+            "AgeStart": "age",
+            "Value": "births_p_1000f",
+        },
+        inplace=True,
+    )
+
+    # Merge in the male and female population by age data
+    fert_rates_df = fert_rates_df.merge(
+        pop_df[["year", "age", "pop"]][pop_df["sex_num"] == 1],
+        how="left",
+        on=["year", "age"],
+    )
+    fert_rates_df.rename(columns={"pop": "pop_male"}, inplace=True)
+    fert_rates_df = fert_rates_df.merge(
+        pop_df[["year", "age", "pop"]][pop_df["sex_num"] == 2],
+        how="left",
+        on=["year", "age"],
+    )
+    fert_rates_df.rename(columns={"pop": "pop_female"}, inplace=True)
+    fert_rates_df["fert_rate"] = fert_rates_df["births_p_1000f"] / (
+        1000 * (1 + (fert_rates_df["pop_male"] / fert_rates_df["pop_female"]))
+    )
+    fert_rates_df = fert_rates_df[
+        (
+            (fert_rates_df["year"] >= start_year)
+            & (fert_rates_df["year"] <= end_year)
+        )
+    ]
+
+    return fert_rates_df
+
+
+def get_un_mort_data(
+    country_id: str = "710",
+    start_year: int = 2021,
+    end_year: int = None,
+    download: bool = True,
+) -> pd.DataFrame:
```

Comment:
```suggestion
    download: bool = True,
    variant: int = 4,
) -> pd.DataFrame:
```
Note: this is only needed for the population data. The deaths data is only reported for the Median variant